scripts/ethbuild.sh

```diff
@@ -223,7 +238,12 @@ do
 			exit 1
 		fi
 
-		make -j${MAKE_CORES}
+		if [[ $USEXCODE -eq 0 ]]; then
+			make -j${MAKE_CORES}
+		else
+			xcodebuild
```

Comment:
We use simple make for C++ building in Macos. It works fine since this is what our buildbot also uses. If this change went in the MacosX build slave we had would fail.